src/spring/azext_spring/commands.py

```diff
@@ -303,6 +321,30 @@ def load_command_table(self, _):
         g.custom_command('unbind', 'api_portal_custom_domain_unbind')
         g.custom_command('update', 'api_portal_custom_domain_update')
 
+    with self.command_group('spring application-accelerator',
+                            custom_command_type=application_accelerator_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_show_command('show', 'application_accelerator_show', table_transformer=transform_application_accelerator_output)
+        g.custom_command('create', 'application_accelerator_create', table_transformer=transform_application_accelerator_output)
+        g.custom_command('delete', 'application_accelerator_delete', supports_no_wait=True)
+
+    with self.command_group('spring application-accelerator predefined-accelerator',
+                            custom_command_type=predefined_accelerator_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('list', 'predefined_accelerator_list', table_transformer=transform_predefined_accelerator_output)                        
+        g.custom_show_command('show', 'predefined_accelerator_show', table_transformer=transform_predefined_accelerator_output)
+        g.custom_command('disable', 'predefined_accelerator_disable', supports_no_wait=True)
+        g.custom_command('enable', 'predefined_accelerator_enable', supports_no_wait=True)
```

Comment:
Can we use create/delete here instead of enbale/disable?

Reply:
No